webapp/src/components/expanded_view/component.tsx

```diff
@@ -155,6 +159,128 @@ export default class ExpandedView extends React.PureComponent<Props, State> {
+                alignItems: 'center',
+                justifyContent: 'center',
+                padding: '16px 8px',
+                width: '100%',
+            },
+            centerControls: {
+                display: 'flex',
+                alignItems: 'center',
+                justifyContent: 'center',
+            },
+            topContainer: {
+                display: 'flex',
+                alignItems: 'center',
+                justifyContent: 'center',
+                fontSize: '16px',
+                lineHeight: '24px',
+                fontWeight: 600,
+                height: '56px',
+                padding: '0 20px',
+            },
+            screenContainer: {
+                display: 'flex',
+                flex: 1,
+                flexDirection: 'column',
+                justifyContent: 'center',
+                alignItems: 'center',
+                maxWidth: 'calc(100% - 16px)',
+                background: 'rgba(var(--button-color-rgb), 0.08)',
+                borderRadius: '8px',
+                margin: '0 12px',
+            },
+            screenSharingMsg: {
+                display: 'flex',
+                alignItems: 'center',
+                gap: '4px',
+                background: 'var(--calls-bg)',
+                padding: '4px 8px',
+                borderRadius: '12px',
+                lineHeight: '12px',
+                fontSize: '10px',
+                fontWeight: 600,
+                letterSpacing: '0.02em',
+                color: 'var(--button-color)',
+                margin: '4px',
+            },
+            rhs: {
+                display: 'flex',
+                flexDirection: 'column',
+                width: '280px',
+                background: 'var(--center-channel-bg)',
+                color: 'var(--center-channel-color)',
+                margin: 0,
+                padding: 0,
+                overflow: 'auto',
+                gap: '10px',
+            },
+            rhsHeaderContainer: {
+                position: 'sticky',
+                top: '0',
+                background: 'var(--center-channel-bg)',
+            },
+            rhsHeader: {
+                display: 'flex',
+                alignItems: 'center',
+                background: 'rgba(var(--center-channel-color-rgb), 0.04)',
+                borderBottom: '1px solid rgba(var(--center-channel-color-rgb), 0.08)',
+                fontFamily: 'Metropolis, sans-serif',
+                fontWeight: 600,
+                fontSize: '16px',
+                padding: '0 16px',
+                height: '63px',
+                lineHeight: '63px',
+            },
+            centerView: {
+                display: 'flex',
+                flex: 1,
+                overflow: 'auto',
+                background: 'rgba(var(--button-color-rgb), 0.08)',
+                borderRadius: '8px',
+                margin: '0 12px',
+            },
+        };
+    };
+
+    private style = this.genStyle();
```

Comment:
Eventually we should decide on a standard -- scss, style objects, styled components, etc. But not now.

Reply:
Agreed. If it wasn't for the little special cases (e.g. `:hover`) I'd be happy to keep doing this but yeah we should review and at least avoid CSS in three different ways. Created https://mattermost.atlassian.net/browse/MM-52387